scripts/test_emscripten.sh

```diff
@@ -29,28 +29,25 @@
 set -e
 
 REPO_ROOT=$(cd $(dirname "$0")/.. && pwd)
-
-cd $REPO_ROOT/build
-
-echo "Preparing solc-js..."
-rm -rf solc-js
-git clone https://github.com/ethereum/solc-js
-cd solc-js
-npm install
-
-# Replace soljson with current build
-echo "Replacing soljson.js"
-rm -f soljson.js
-# Make a copy because paths might not be absolute
-cp ../solc/soljson.js soljson.js
-
-# Update version (needed for some tests)
-VERSION=$(../../scripts/get_version.sh)
-echo "Updating package.json to version $VERSION"
-npm version $VERSION
-
-echo "Running solc-js tests..."
-npm run test
+SOLJSON="$REPO_ROOT/build/solc/soljson.js"
+
+DIR=$(mktemp -d)
+(
+    echo "Preparing solc-js..."
+    git clone --depth 1 https://github.com/ethereum/solc-js "$DIR"
+    cd "$DIR"
+    npm install
+    cp "$SOLJSON" soljson.js
```

Comment:
I think it should have `rm -f soljson.js` in case it is a link or something is messed up (I think it was needed at some point). Also please add back the "replacing soljson" line.

Reply:
There is no `soljson.js` in https://github.com/ethereum/solc-js repo. And this is a clean clone.

Reply:
`npm install` calls `prepublish`, which calls `downloadsolc` and that downloads it :)

Reply:
Will do, thanks.